pkg/lib/cockpit-rsync-plugin.js

```diff
@@ -0,0 +1,31 @@
+const child_process = require("child_process");
+
+module.exports = class {
+    constructor(options) {
+        if (!options)
+            options = {};
+        this.dest = options.dest;
+        if (options.source) {
+            this.source = options.source;
+        } else {
+            this.source = "dist/";
+        }
```

Comment:
Usual JS syntax for defaults: `this.source = options.source || "dist/";`

Reply:
:+1: